src/app/cluster/cluster-details/machine-deployment-details/template.html

```diff
@@ -29,6 +29,12 @@
       <div fxFlex></div>
       <div fxLayout="row"
            fxLayoutGap="10px">
+        <button mat-icon-button
+                color="tertiary"
+                (click)="showRestartDialog()"
+                [disabled]="!isEditEnabled()">
+          <i class="km-icon-reset"></i>
```

Comment:
Not sure about this one, and we can also wait for @cschieder to be back, but maybe `km-icon-regenerate` would be a better fit? 
We're already using it in the serviceaccounts > tokens to regenerate the token. Regenerate for me feels more like a restart, than reset 🤔 But maybe I'm wrong 😅

Reply:
This is a valid point. Actually, to me the `reset` icon looks like a `Go back` button, not a real `reset` :slightly_smiling_face:.
I'd go with the `regenerate` icon also.

Reply:
True. I didn't remember about this icon. Fixed.